src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.Session_FilterModel.cs

```diff
@@ -95,29 +86,29 @@ private CompletionItem GetCompletionItem(CompletionItem item)
                 }
 
                 var textSnapshot = caretPosition.Snapshot;
-                var allFilteredItems = new List<CompletionItem>();
+                var allFilteredItems = new List<PresentationItem>();
                 var textSpanToText = new Dictionary<TextSpan, string>();
-                var completionRules = _completionRules;
+                var completionRules = this.Controller.GetCompletionHelper(model);
```

Comment:
one of these names seems wrong.  this should either be "var completoinHelper" or "GetCompletionRules".